CMSgov/easi-app
# [EASI-2715, EASI-2758] - Admin notes form, note box, and note modal

# EASI-2715
# EASI-2758

[Figma](https://www.figma.com/file/iGatgTnlzTuqW9CQFYHDam/TRB-Intake-e2e?node-id=205-17248&t=rEo94khHyIqjfBr1-0)
[Figma - Modal](https://www.figma.com/file/iGatgTnlzTuqW9CQFYHDam/TRB-Intake-e2e?node-id=1015-72256&t=pxILOPDTih7lt8Ba-0)

## Changes and Description

- Added Admin `Notes` view
  - `Note` component
- Updated/integrated `NoteBox`
- Added form `AddNote`

![Screenshot 2023-03-24 at 1 54 08 PM](https://user-images.githubusercontent.com/95709965/227603078-05ff3c6a-996b-4dab-886b-7eb3f681c552.png)

![Screenshot 2023-03-24 at 1 54 22 PM](https://user-images.githubusercontent.com/95709965/227603141-d4305651-451a-41d5-9268-e9b019529dbb.png)

![Screenshot 2023-03-24 at 1 53 42 PM](https://user-images.githubusercontent.com/95709965/227603005-b1aaa28b-486f-45d0-a2f6-c0ef5248a79f.png)

![Screenshot 2023-03-24 at 1 54 41 PM](https://user-images.githubusercontent.com/95709965/227603204-6da43480-1e31-4a89-98f3-b2e5d0d8e134.png)

## How to test this change

- Create TRB request and navigate to admin view
- View notes
- Add notes
- Open modal from NoteBox component on various admin view
- View/Add notes from within modal
- Verify against Figma

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
